core/controlplane/config/templates/cloud-config-controller

```diff
@@ -2380,7 +2380,7 @@ write_files:
                   - --namespace=kube-system
                   - --configmap=kube-dns-autoscaler
                   - --target=Deployment/kube-dns
-                  - --default-params={"linear":{"coresPerReplica":256,"nodesPerReplica":16,"min":2}}
+                  - --default-params={"linear":{"coresPerReplica":{{ .kubeDns.autoscaling.coresPerReplica }},"nodesPerReplica":{{ .kubeDns.autoscaling.nodesPerReplica }},"min":{{ .kubeDns.autoscaling.min}}}}
```

Comment:
I guess `kubeDns` should be capitalized so that it is `KubeDns`.

That would be the cause of failures seen in the travis build:

```
--- FAIL: TestRenderStackTemplate (0.11s)
	Error Trace:	cluster_test.go:768
			helper.go:100
			helper.go:22
			cluster_test.go:772
	Error:      	Received unexpected error:
	            	failed to render controller cloud config: template: ../config/templates/cloud-config-controller:2383:77: executing "s3" at <.kubeDns.autoscaling...>: can't evaluate field kubeDns in type *config.Config
	Messages:   	Unable to initialize Cluster
```
https://travis-ci.org/kubernetes-incubator/kube-aws/builds/339246310